cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -405,13 +408,16 @@ func execTrivy(opt options, action string, actionArgs []string) (*results, error
 
 	logger.Infof("running command: %s %s\n", trivyCmd, trivyArgs)
 
+	var inErr error
 	err := retry.Do(
```

Comment:
It's not from this PR but.
Do we need a retry? or a retry of 3 times? 
I think most of the checks doesn't have a retry mechanism in place.

Reply:
I thought the same when I saw it. Maybe @danfaizer can bright more light here.

Reply:
As far as I remember, when I developed the check long ago, if the private registry was returning an error (for example 429 or 503) and wanted the check to retry.
However, now we know if the target is reachable or not (INCONCLUSIVE), therefore maybe we can remove the retry logic and in case we detect an increase of FAILED checks take action in the underlaying case of the error instead of just try to retry.

TL;DR
Perhaps yes: is better to remove the retry logic and log the Trivy command output accordingly in the error section of the report.

Reply:
If you agree let's do one thing: I'll merge the PR as it is now as it will allow us to debug better the error causes. Then, I'll create a new PR removing the retry and if after the next round of scans we see an increase of errors related to network, we can just revert that commit. WDYT @jesusfcr @danfaizer?

Reply:
sure